easybuild/easyconfigs/c/CRISPR-DAV/CRISPR-DAV-2.3.4-foss-2020b.eb

```diff
@@ -0,0 +1,35 @@
+name = 'CRISPR-DAV'
+version = '2.3.4'
+
+homepage = 'https://github.com/pinetree1/crispr-dav/'
+description = """CRISPR-DAV is a pipeline to analyze
+amplicon-based NGS data of CRISPR clones in a high throughput manner."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/pinetree1/crispr-dav/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['crispr-dav-2.3.4-foss2020b_remove_hardcoding.patch']
+checksums = [
+    '49975cd48bdbf31fe5a9e2aaa3f5ed85d3cc6f65a422ee3aa8daed890159d2ae',  # v2.3.4.tar.gz
+    # crispr-dav-2.3.4-foss2020b_remove_hardcoding.patch
+    '41475a09754d65cc2c88d4161be6eacdfc98242bccbe49778c4f1f74210a6cb2',
+]
+
+dependencies = [
+    ('R', '4.0.3'),
+    ('Perl', '5.32.0'),
+    ('NGS', '2.10.9'),
+    ('Pysam', '0.16.0.1'),
+    ('Excel-Writer-XLSX', '1.09'),
+    ('naturalsort', '0.1.3'),
```

Comment:
I think it makes more sense to install `naturalsort` as an additional extension in the `R` module, since it's a standard `R` package, see #13762

Reply:
so should I remove it altogether from here?
I see it has been added to `R` already

Reply:
you answered elsewhere, removing